### DISCUSSION TITLE
[Feature Request]: IPv6 support in Linux Native

### DISCUSSION THREAD

**Discussion by njh** [meshtastic/firmware#4986] [Ideas]
### Platform

Linux Native

### Description

When I checked the meshtasticd process running on my Raspberry Pi, I noticed that it was only binding to IPv4 and not IPv6.

```
$ sudo lsof -ni
meshtasti 2678  root   15u  IPv4  12080      0t0  TCP *:https (LISTEN)
meshtasti 2678  root   16u  IPv4  12081      0t0  TCP *:4403 (LISTEN)
```

Please can you also bind to IPv6?

**Comment by cvmiller** [comment]
Yes, please update meshtasticd to listen on an IPv6 socket as well. 

With 50% of the internet traffic being carried over  IPv6 (as measured by Google), and IPv6 only networks (I have one at my house), listening to an IPv6 socket is even more important.
